config/initializers/decidim.rb

```diff
@@ -646,3 +646,6 @@
     }
   ]
 end
+
+# Workaround until v0.28.0 or v0.28.0.rc3 is available
```

Comment:
If i remember correctly, the fix will be in v0.28.0.rc4 ?

Reply:
Nop, it was in rc3. This workaround is no longer needed, nice catch 👏🏽 👏🏽